### ISSUE TITLE
v2 support

### ISSUE THREAD

**Issue by maresb** [conda-forge/aws-cdk.core-feedstock#1]
### Comment:

Hey @ocefpaf, is aws-cdk v2 on conda-forge? If so, I haven't been able to locate it. Thanks!

**Comment by ocefpaf** [comment]
I don't think we have it. To be honest, I don't use these packages. I created them for someone else.

**Comment by maresb** [comment]
Ok, thanks @ocefpaf! I won't be taking this on; I'm just going to install from PyPI.

In case anyone else finds this and is looking for orientation, it seems that the zoo of v1 packages has been consolidated into the single [aws-cdk-lib](https://pypi.org/project/aws-cdk-lib/)

**Comment by ocefpaf** [comment]
> it seems that the zoo of v1 packages has been consolidated into the single [aws-cdk-lib](https://pypi.org/project/aws-cdk-lib/)

Awesome! It was quite complicated to package all these interconnected packages. If the parts have a life on their own, sure, it makes sense. I'm not sure if that is true though and a aws-cdk-lib probably makes a lot of sense.

**Comment by maresb** [comment]
> It was quite complicated to package all these interconnected packages. If the parts have a life on their own, sure, it makes sense. I'm not sure if that is true though and a aws-cdk-lib probably makes a lot of sense.

I'm new to AWS CDK, so I'm still figuring things out for myself. That said, my impression is that the AWS development team also came to the same conclusion as you: that having all these interconnected packages was not worth the effort, and hence switched to the aws-cdk-lib approach for v2.
